awx/settings/defaults.py

```diff
@@ -406,19 +403,13 @@ def IS_TESTING(argv=None):
 # The time (in seconds) between the periodic isolated heartbeat status check
 AWX_ISOLATED_PERIODIC_CHECK = 600
 
-# Verbosity level for isolated node management tasks
-AWX_ISOLATED_VERBOSITY = 0
-
 DEVSERVER_DEFAULT_ADDR = '0.0.0.0'
 DEVSERVER_DEFAULT_PORT = '8013'
 
 # Set default ports for live server tests.
 os.environ.setdefault('DJANGO_LIVE_TEST_SERVER_ADDRESS', 'localhost:9013-9199')
 
-BROKER_DURABILITY = True
-BROKER_POOL_LIMIT = None
```

Comment:
This predates me, looks very old, and I don't see any evidence of its use these days.